opengrok-indexer/src/main/java/org/opengrok/indexer/web/Scripts.java

```diff
@@ -86,27 +89,27 @@ public FileScript(String script, int priority) {
 
         @Override
         public String toHtml() {
-            StringBuilder builder = new StringBuilder();
-            builder.append("\t<script type=\"text/javascript\" src=\"");
-            builder.append(this.getScriptData());
-            builder.append("\" data-priority=\"");
-            builder.append(this.getPriority());
-            builder.append("\"></script>\n");
-            return builder.toString();
+            return "\t<script type=\"text/javascript\" src=\"" +
+                    this.getScriptData() +
+                    "\" data-priority=\"" +
+                    this.getPriority() +
+                    "\"></script>\n";
         }
 
     }
 
     protected static final Map<String, Script> SCRIPTS = new TreeMap<>();
 
+    private static final WebJarAssetLocator assetLocator = new WebJarAssetLocator();
+
     /**
      * Aliases for the page scripts. The path in the FileScript is relatively to
      * the request's context path.
      *
      * @see HttpServletRequest#getContextPath()
      */
     static {
-        putjs("jquery", "js/jquery-3.4.1", 10);
+        putFromWebJar("jquery", "jquery.min.js", 10);
```

Comment:
I wonder if this is going to work well w.r.t. caching. So far we relied on the version number to change I think.

Reply:
The final code looks like this so version is still there :)
```html
<script type="text/javascript" src="/opengrok_web_war/webjars/jquery/3.5.1/jquery.min.js" data-priority="10"</script>
```

Reply:
Yes, caching.

Reply:
I'll probably need to modify `ExpiresOneDayFilter` to include `/webjars`.

Reply:
Then it would be ok I guess.